package.json

```diff
@@ -71,8 +71,8 @@
     "clean": "rm -rf libcsm/ libesm/ dist/ coverage/ .nyc_output/",
     "lint": "yarn eslint",
     "build": "yarn lint && yarn generate-esm && yarn generate-cjs && yarn check-types && yarn run-rollup && yarn run-uglify",
-    "generate-cjs": "yarn tsc --module commonjs --outDir libcjs && echo '{\"type\": \"commonjs\"}' > libcjs/package.json",
-    "generate-esm": "yarn tsc --module nodenext --outDir libesm && echo '{\"type\": \"module\"}' > libesm/package.json",
+    "generate-cjs": "yarn tsc --module commonjs --outDir libcjs && echo {\"type\": \"commonjs\"} > libcjs/package.json",
```

Comment:
I am not *sure* if it'll serve a purpose, but I figure we should add `"sideEffects": false` here too? It's true and could *plausibly* be useful, since a quick google reveals that tree-shaking plugins for CommonJS *do* exist. (I have no idea whether they respect `"sideEffects": false`, but even if they don't today, they might do tomorrow.)

Reply:
I didn't add it since Prettier only use ESM version 😄